object-api-multipart.go

```diff
@@ -243,14 +236,15 @@ func (o objectAPI) ListMultipartUploads(bucket, prefix, keyMarker, uploadIDMarke
 		}).Errorf("listMetaVolumeFiles failed with %s", e)
 		return ListMultipartsInfo{}, probe.NewError(e)
 	}
+
+	// Loop through all the received files fill in the multiparts result.
 	for _, fi := range fileInfos {
 		var objectName string
 		var uploadID string
 		if fi.Mode.IsDir() {
+			// All directory entries are common prefixes.
 			objectName = strings.TrimPrefix(fi.Name, retainSlash(bucket))
-			// For a directory entry
 			result.CommonPrefixes = append(result.CommonPrefixes, objectName)
-			continue
```

Comment:
i think just removing this `continue` statement will fix the problem, because if we do continue here the result.NextUploadIDMarker will never be reset to `""` and it will always have some uploadID